.babelrc

```diff
@@ -0,0 +1,5 @@
+{
+  "plugins": [
+    "istanbul"
```

Comment:
I'm not sure what effect adding this file will have on the existing babel config. Coverage didn't work until I added this though.

Reply:
We already have a `babel.config.js` file which specifies `plugins: ['istanbul']`. Is this file required?

Reply:
All I know is that it didn't work until I added this...

Reply:
Interesting. It could be confusing for new developers coming to the project to see two babel config files with a duplicated plugins config. I'm assuming that whatever generates the Vue code coverage is unable to read the `babel.config.js` file, but is able to read the `.babelrc` file.

Looking at the babel docs, at https://babeljs.io/docs/en/configuration#babelconfigjs, it has `babel.config.js` set up to export a function. If you change it to use that format, does it work?

Reply:
Thanks, changing to that format seems to work.